go/vt/vtgate/vtgate.go

```diff
@@ -82,6 +82,8 @@ var (
 	// lockHeartbeatTime is used to set the next heartbeat time.
 	lockHeartbeatTime = flag.Duration("lock_heartbeat_time", 5*time.Second, "If there is lock function used. This will keep the lock connection active by using this heartbeat")
 	warnShardedOnly   = flag.Bool("warn_sharded_only", false, "If any features that are only available in unsharded mode are used, query execution warnings will be added to the session")
+
+	foreignKey = flag.String("foreign_key", "allow", "This is to provide how to handle foreign key constraint in create/alter table. Valid values are: allow, disallow")
```

Comment:
@systay and @shlomi-noach can do the substantive review.
My comment is on the flag name. `foreign_key_mode` may be more descriptive and useful.